Zujayli blog

zainaujayli opened the pull request:
Hello Brandon, 

Here is the completed blog! It looked alright in preview but I'd appreciate you taking a quick glance to make sure I didn't screw anything up in the code ( I learned some footnotes from Mikki - those are the only things looking strange on my end). Thank you so much for all your help with this. I hope you have a good week. 

Best, 
Zaina

walshbr commented:
got it and published it! @zainaujayli - a couple things:

1) the date on the post is set from the filename and metadata at the top, not the date we actually publish it. So in the future you'll want to adjust the filename so that it takes the current date. Otherwise it will appear to have been published in the past, with older posts appearing more recent than it. I updated it on this one those.
2) you should use relevant links for the images - so 

/assets/post-media/2020-10-01-a156T.jpg

instead of

https://github.com/scholarslab/scholarslab.org/blob/zujayli-blog/assets/post-media/2020-10-01-a156T.jpg

But I fixed both these and published! Just for the future things to keep in mind.

zainaujayli commented:
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
/* List Definitions */
@list l0
	{mso-list-id:697001396;
	mso-list-template-ids:-1;}
ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->Hello Brandon,  Thanks for the clarifying notes and all the help! I really appreciate it.  Have a wonderful weekend.  Best,Zaina From: Brandon WalshSent: Wednesday, October 7, 2020 8:16 AMTo: scholarslab/scholarslab.orgCc: zainaujayli; MentionSubject: Re: [scholarslab/scholarslab.org] Zujayli blog (#895) got it and published it! @zainaujayli - a couple things:the date on the post is set from the filename and metadata at the top, not the date we actually publish it. So in the future you'll want to adjust the filename so that it takes the current date. Otherwise it will appear to have been published in the past, with older posts appearing more recent than it. I updated it on this one those.you should use relevant links for the images - so/assets/post-media/2020-10-01-a156T.jpginstead ofhttps://github.com/scholarslab/scholarslab.org/blob/zujayli-blog/assets/post-media/2020-10-01-a156T.jpgBut I fixed both these and published! Just for the future things to keep in mind.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. 
